src/migrations/embedded-store-to-fs-store.js

```diff
@@ -331,6 +339,12 @@ function migrateImages(basePath) {
         $$date: getTimestampFromNedbDate(dataFile.content.modificationDate)
       };
 
+      let imageContent = dataFile.content.content;
+      if (typeof imageContent === 'object') {
```

Comment:
i think that the check will need to be `typeof imageContent === 'object' && !Array.isArray(imageContent)` becasuse `typeof` returns `object` for arrays too :laughing: